### PR TITLE
chore: update cypress version to 4.9.0

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* update cypress version to `4.9.0`

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: packages/toast-ui.grid/cypress/integration/dataSource.spec.ts [hunk 1]
```diff
@@ -574,48 +575,46 @@ it('stop custom event if prev event is prevented.', () => {
...
-//     setTimeout(() => {
-//       cy.getByCls('body-container')
-//         .invoke('height')
-//         .then(height => {
-//           initialHeight.should('be.eq', height);
-//         });
-//     });
-//   });
-// });
+function createGridWithScrollType(dataSource?: DataSource) {
```
Comment (jung-han) [inline packages/toast-ui.grid/cypress/integration/dataSource.spec.ts:578]: 크..좋군요

File: packages/toast-ui.grid/cypress/integration/clientPagination.spec.ts [hunk 2]
```diff
@@ -207,13 +207,11 @@ describe('type: scroll', () => {
 
     cy.gridInstance().invoke('setPerPage', 30);
 
-    setTimeout(() => {
```
Comment (seonim-ryu) [inline packages/toast-ui.grid/cypress/integration/clientPagination.spec.ts:210]: 이건 cypress 버전 업데이트로 없어진건가요??

File: packages/toast-ui.grid/cypress/integration/index.spec.ts [hunk 3]
```diff
@@ -20,15 +20,15 @@ it('observe() should invoke callback function whenever related props changed', (
   expect(callback1).to.be.calledWith('Kim', 'JS Guide');
   expect(callback2).to.be.calledWith(20, 'Lee');
 
-  callback1.reset();
-  callback2.reset();
+  callback1.resetHistory();
```
Comment (seonim-ryu) [inline packages/toast-ui.grid/cypress/integration/index.spec.ts:23]: 네이밍이 변경되었군요ㅎ 기능 명시가 더 확실해진것 같네요ㅎㅎ 👍

File: packages/toast-ui.grid/cypress/integration/clientPagination.spec.ts (hunk 2 above)
Comment (js87zz) [reply packages/toast-ui.grid/cypress/integration/clientPagination.spec.ts:210]: 넵 타이밍 이슈땜에 넣었던건대 이제는 잘 돌아가네요ㅎㅎ

**Pull request merged**